core/src/main/java/org/eclipse/hono/util/RegistryManagementConstants.java

```diff
@@ -107,6 +107,10 @@
     public static final String FIELD_CREDENTIALS_TOTAL           = "total";
 
     /* secrets fields */
+    /**
+     * The name of the field that contains the secret id.
+     */
+    public static final String FIELD_SECRETS_ID                 = "id";
```

Comment:
How about renaming this to the more general `FIELD_ID` since it is not really secrets specific anyway and might be used for other entities as well. WDYT?

Reply:
Why not. I updated the description so it doesn't lower the code readability.